charts/neo4jv5-internal/values.yaml

```diff
@@ -49,7 +49,11 @@ neo4j:
       ]
 
   env:
-    NEO4JLABS_PLUGINS: '["apoc"]'
+    NEO4J_PLUGINS: '["apoc"]'
```

Comment:
because i was seeing in logs name is deprecated:
	
2024-01-31 10:56:01.269	
The old name will still work, but is likely to be deprecated in future releases.
	
2024-01-31 10:56:01.269	
NEO4JLABS_PLUGINS has been renamed to NEO4J_PLUGINS since Neo4j 5.0.0.

Reply:
yes, thx